src/develop/ui/patterns/mobile/datepicker.md

```diff
@@ -3,68 +3,79 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# DatePicker Pattern
+# Date Picker
 
-The DatePicker pattern provides you with a date and time picker with a flat UI to display inline on the screen. It can receive lists of dates with events and it enables you have a selection within a range of days. The DatePicker was created using the [Pikaday.js
-library](https://github.com/dbushell/Pikaday/blob/master/README.md "https://github.com/dbushell/Pikaday/blob/master/README.md") .
+You can use the Date Picker UI pattern as a graphical interface widget that allows the user to select a date from a calendar and/or time from a time range.
+
+To display items in a gallery pattern, you can use static data or a **List** widget inside the block. 
```

Comment:
a) What's a "gallery" pattern?
b) Does the "static data or a List widget" mention belong here, or to the "How to use" section?